### DISCUSSION TITLE
SI Stone Enclosure Easter Egg

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#364] [Show and tell]
In SI, it is possible to enter the frozen north without going through Gorlab. You will need a spellbook with Fetch in the 8th Circle, and powder kegs from Spektor's cave.

Once you return to the mainland from Moonshade, go to the area north of Furnace with the ruined mansion and the stone enclosure north of it. Northeast of that, there's a lot of wood. You'll want as many vertical orientated ones as your party can carry.

Walk to the one spot along the stone enclosure where you can throw things into the enclosure. In DOSbox this is easy, but if you have trouble in Exult try killing a sheep when they are close to the wall to get in.

![image](https://github.com/exult/exult/assets/117867844/7b8bf699-a97e-4ffa-a9ca-b0c18e5dedf7)

![image](https://github.com/exult/exult/assets/117867844/63bd65bd-c20f-4297-a89b-e0801620bb8a)

Once side, make your own staircase. This is similar to the process in the Mountains of Freedom. Now you can walk above the walls, and even over closed doors. Go clockwise (not the way shown here).

![image](https://github.com/exult/exult/assets/117867844/543cbf80-be6b-41e1-87ea-45bd52acbe47)

You will come across this shining green key (Quality 115). Cast Fetch on it.

![image](https://github.com/exult/exult/assets/117867844/123bfcb7-5fcb-4985-adfc-86eb2e32af3c)

Go back to the teleport area and use the key on the brass doors there. You are now inside the Shrine of Chaos.

![image](https://github.com/exult/exult/assets/117867844/11d3b132-8b1e-4f3c-88b1-eaf7417b9f2c)

The staircase nearby will lead you to the huge brass doors. A map is available here:
https://strategywiki.org/wiki/File:Ultima_VII_-_SI_-_Shrine_of_Chaos.png

The only way I've found to crack these doors is to kill a companion (an automaton) and stuff a powder keg inside his body. This allows the blast to be close enough to trigger the relevant egg.

![image](https://github.com/exult/exult/assets/117867844/f8f3cd2b-235c-45b4-ae56-27a4bddce62b)

Now you are free to slay Vasculio and use the key on his corpse to head towards Spinebreaker.

![image](https://github.com/exult/exult/assets/117867844/92577088-b3e3-4509-a7d6-d5cc49f19358)

I don't think this will break the game since Gorlab to the Gwani are not immediately plot critical.

**Comment by KnightCaptainU7** [comment]
I should note you can also use powder kegs from the Silver Seed area.

**Comment by DominusExult** [comment]
:)
